### PR TITLE
sys/docs: at_client documentation

### PR DESCRIPTION
**Pull request by RocioRojas**
### Contribution description
Documentation for the at_client module was written

### Testing procedure
Look at firmware/sys/at_client/doc.txt

### CODE REVIEW

File: firmware/sys/at_client/doc.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
+@ingroup     sys
+
```
Comment (luisan00) [inline firmware/sys/at_client/doc.txt:3]: Define an unique doxygen group for this component

File: firmware/sys/at_client/doc.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# What goes here?
```
Comment (luisan00) [inline firmware/sys/at_client/doc.txt:4]: 1st level title should be unique for a component.

**Review by luisan00 — CHANGES_REQUESTED** [review]
Some changes needs to be addressed

File: firmware/sys/at_client/include/at_client.h [hunk 3]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 /**
- * @ingroup     m4a-firmware
+ * @ingroup     sys
```
Comment (luisan00) [inline firmware/sys/at_client/include/at_client.h:17]: This should be something like `AT client` but not sys

File: firmware/sys/at_client/doc.txt (hunk 1 above)
Comment (eduazocar) [inline firmware/sys/at_client/doc.txt:3]: or something like this
```suggestion
@defgroup at_uart_firm
```
*** Be careful to the group defined is not used in wifi-subsys ***

**Review by eduazocar — COMMENTED** [review]
Everything here looks good for me, do squash :+1:

**Review by eduazocar — APPROVED** [review]
Approved :+1:

**Review by eduazocar — APPROVED** [review]
:+1:

**Review by xkevin190 — APPROVED** [review]

**Review by luisan00 — APPROVED** [review]
good job,going ahead!

**Pull request merged**